Tests/RelativityTests/SubviewDistributionTests.swift

```diff
@@ -169,10 +169,10 @@ class SubviewDistributionTests: XCTestCase {
 
             let a = UILabel()
             a.text = "Some text"
-            a.sizeToFit()
+            a.bounds.size = CGSize(width: 80, height: 25)
```

Comment:
`sizeToFit()` uses the current device's screen proportions, so we need to hard-code the size of the bounds rather than relying on `sizeToFit()` in this mocked-out `screenScale` environment.